DigitalLearningSolutions.Data/Services/CentreCustomPromptsService.cs

```diff
@@ -63,6 +65,18 @@ public CentreCustomPrompts GetCustomPromptsForCentreByCentreId(int centreId)
             );
         }
 
+        public IEnumerable<CustomPrompt> GetCustomPromptsThatHaveOptionsForCentreByCentreId(int centreId)
+        {
+            var result = centreCustomPromptsDataService.GetCentreCustomPromptsByCentreId(centreId);
+
+            var centreCustomPrompts = new CentreCustomPrompts(
+                result.CentreId,
+                PopulateCustomPromptListFromCentreCustomPromptsResult(result)
+            );

```

Comment:
Can we not just call the other service method and then do the where on it?

Reply:
Yep, changed